docs/how-to/python_api.md

```diff
@@ -0,0 +1,188 @@
+## Use the Spark Client from Python 
+
+In the following, we show how to use the spark client to manage service accounts
+using Python.
+
+### Bind to Kubernetes
+
+The following snipped allows you to import relevant environment variables
+into a confined object, among which there should an auto-inference of your 
+kubeconfig file location. 
+
+```python
+import os
+from spark_client.domain import Defaults
+from spark_client.services import KubeInterface
+
+# Defaults for spark-client
+defaults = Defaults(dict(os.environ))  # General defaults
+
+# Create a interface connection to k8s
+kube_interface = KubeInterface(defaults.kube_config)
+```
+
+Notat that if you want to override some of these settings,
+you can extend the `Default` class accordingly. 
+
+Alternatively you can also use auto-inference using the `kubectl` command via
+
+```python
+from spark_client.services import KubeInterface
+
+kube_interface = KubeInterface.autodetect(kubectl_cmd="kubectl")
+```
+
+Once you have binded to the k8s cluster, you have some 
+properties of the connection readily available, e.g. 
+
+```python
+kube_interface.namespace
+kube_interface.api_server
+```
+
+You can also issue some `kubectl` commands, using the `exec` method
```

Comment:
Again, this is specific to kubectl based implementation, which also needs a proper naming if we want to include this as well apart from 'LightKube' version.